fe/fe-core/src/test/java/com/starrocks/sql/plan/LowCardinalityTest.java

```diff
@@ -120,6 +120,65 @@ public static void beforeClass() throws Exception {
                 "\"replication_num\" = \"1\",\n" +
                 "\"in_memory\" = \"false\"\n" +
                 ");");
+        starRocksAssert.withTable("CREATE TABLE `low_card_t1` (\n" +
```

Comment:
could we simplify the case?

Reply:
I try use the exist table to rewrite the case, but failed....